tests/skydb_integration_test.go

```diff
@@ -0,0 +1,278 @@
+package tests
+
+import (
+	"bytes"
+	"github.com/NebulousLabs/go-skynet/v2"
+	"gopkg.in/h2non/gock.v1"
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+// TestGetJSON tests get of JSON from registry.
+func TestGetJSON(t *testing.T) {
+	defer gock.Off() // Flush pending mocks after test execution
+
+	const srcFile = "../testdata/jsonFile1.json"
+	const skylink = "AADeqJt8vPZtW9Nm_Hc5C5EKPmZhYUZGeBqvibofIMMHtg"
+
+	registryFile, err := ioutil.ReadFile(srcFile)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	opts := skynet.DefaultDownloadOptions
+	urlpath := strings.TrimRight(opts.EndpointPath, "/") + "/" + skylink
```

Comment:
nit: I believe we have a `EnsureSuffix` helper in Sia that may be useful here.

Reply:
`modules.EnsureSuffix` :+1: